internal/pkg/template/upgrade/context.go

```diff
@@ -20,9 +20,9 @@ type Context struct {
 	*use.Context
 }
 
-func NewContext(ctx context.Context, templateRef model.TemplateRef, objectsRoot filesystem.Fs, instanceId string, targetBranch model.BranchKey, inputsValues template.InputsValues, inputsDefs map[string]*template.Input, tickets *storageapi.TicketProvider, projectState *state.State) *Context {
+func NewContext(ctx context.Context, templateRef model.TemplateRef, objectsRoot filesystem.Fs, instanceId string, targetBranch model.BranchKey, inputsValues template.InputsValues, inputsDefs map[string]*template.Input, tickets *storageapi.TicketProvider, components *model.ComponentsMap, projectState *state.State) *Context {
```

Comment:
Prvy commit je pridanie `components` do template `use/upgrade` context.
- Tato struktura nesie v sebe vsetko potrebne pre vykonanie sablony.
- Tych parametrov je tu dost, ale neda sa to jednoducho zlucit a nemozem na tom teraz palit cas.